src/components/etProvider/EtOverlayProvider.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <slot></slot>
+    <div
+        class="et-sdk-overlay-provider"
+        :class="{
+            'et-sdk-overlay-provider__is-transparent': isTransparent
```

Comment:
Dit is officieel geen BEM, dit is een modifier dus `et-sdk-overlay-provider--is-transparent`

Reply:
Correct, ik heb het dus precies anders om in mijn project. Weet niet of ik het zo wil houden of het hele project wil fixen